source/styleEditor.js

```diff
@@ -1,3 +1,18 @@
 $(document).ready(function(){
-  // your code goes here.
-});
+  $('form').on('submit', function(event) {
+    event.preventDefault()
```

Comment:
since you are already listening to the submit event here this is where I would call `stylePage()` then you don't need the rest of the calls in doc.ready